gamemodes/horde/entities/entities/horde_skull_token/cl_init.lua

```diff
@@ -4,4 +4,10 @@ function ENT:Draw()
     self:DrawModel()
 end
 
-ENT.ColorModulation = Color(1, 0, 1)
\ No newline at end of file
+ENT.ColorModulation = Color(1, 0, 1)
+
+if CLIENT then
```

Comment:
cl_init only runs on client you don't need the CLIENT check, same for SERVER in init.lua

Reply:
oh, k
also gotta love how github doesn't update the conversation tab live and you have to refresh it to see new messages 💀

Reply:
changed btw